docker-compose.yml

```diff
@@ -107,13 +107,13 @@ services:
     container_name: test_game
     environment:
       - REDIS_ADDR=redis:6379
-      - NAMESPACE=TESTGAME
+      - CARDINAL_NAMESPACE=TESTGAME
       - SHARD_SEQUENCER_ADDR=chain:9601
       - BASE_SHARD_ADDR=chain:9090
       - ENABLE_ADAPTER=${ENABLE_ADAPTER:-true}
     build:
-      context: internal/e2e/tester/cardinal
-      dockerfile: Dockerfile
+      context: ./
+      dockerfile: internal/e2e/tester/game/Dockerfile
     restart: unless-stopped
```

Comment:
this keeps the directory structure of the project exactly the same in the docker instance. 

The reason why we do this is because the project now references local packages by using paths relative to the go.mod file. Thus if we change the directory structure in the docker the go.mod will be referencing wrong paths inside the docker. 

Additionally the game docker has been changed such that the context is the root project directory. It can only now be run in the context of the docker-compose from the root.